todo/src/reducers/index.js

```diff
@@ -0,0 +1,16 @@
+import ADD_NEW_TODO from '../actions/index';
+
+const initialState = {
+    input : '',
+}
+
+const addNewTodo = (state = initialState, action) => {
+    switch(action.type){
+        case ADD_NEW_TODO : return{...state, input : action.payload};
```

Comment:
you need to be adding the todo to an array of todos on state (so you'll need to add that to state as mentioned above). Then remember to return an updated state with the new todo added onto the state's todo list. Remember to copy and not mutate as well